tcms/testplans/templates/testplans/get.html

```diff
@@ -80,10 +80,12 @@ <h2 class="card-pf-title">
     <div class="row row-cards-pf">
         <div class="col-xs-12 col-sm-12 col-md-12">
             <div class="card-pf card-pf-accented">
-                <h2 class="card-pf-title">
-                    {% trans 'Confirmed test cases' %}
-                </h2>
-                <div id="confirmed-testcases" class="list-group list-view-pf list-view-pf-view"></div>
+                <div class="card-pf-title">
+                        {% trans 'Confirmed test cases' %}
+                </div>
+                <div class="card-pf-body">
+                    <div id="confirmed-testcases" class="list-group list-view-pf list-view-pf-view"></div>
```

Comment:
In tc_executions.html this is `<div class="list-group tree-list-view-pf">` but I'm not sure what the difference is and if that even matters. The diff here comes from the fact that you've changed the title.

Reply:
I added <div class="card-pf-body"> I will try to use the same css classes as the tc page